include/kernels/coarse_op_kernel.cuh

```diff
@@ -199,7 +204,7 @@ namespace quda {
      @param[in] nFace Depth of the halo
      @param[in] arg Kernel argument
   */
-  template <typename Arg> constexpr bool isHalo(const int coord[], int dim, int nFace, const Arg &arg)
+  template <typename Arg> constexpr bool isHalo(const int coord[], int dim, const int nFace, const Arg &arg)
```

Comment:
Not sure what the point is for this `const` addition for passing an integer by value.

Reply:
Was probably just in a `const` happy mood

https://github.com/lattice/quda/pull/1221/commits/494c0be0466700edb2329aeac14ced9a5ea7e9dd